### PR TITLE
LPS-98315 Consolidate change lists indicator code

### PR DESCRIPTION
**Pull request by stsquared99**
Hi Chema, would you be able to take a quick look to see if our approach makes sense? We are trying to get rid of the individual modules for the control menu entry, as well as use localized css rather than the theme contributor. Feel free to forward if needed. Thanks!

### CODE REVIEW

**Comment by stsquared99** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Hey @stsquared99, I thought you mentioned you were getting rid of Soy, but then this looks to go in a different direction... I'll try to take a look throughout the day. Could you please add some additional context as to why you're doing this and what's the end goal so I can properly assess the approach?

**Comment by stsquared99** [comment]
Hi @jbalsas, sorry for the confusion. Our goal will eventually be to remove soy. Currently we are not yet doing so until our backend rewrite is more solidified.
The reason for this specific change is simply to consolidate our codebase. The code is split into too many individual modules than is necessary, so we are just trying to reduce the number of modules in our component. Let me know if you need further info.
Thanks!

File: modules/apps/change-tracking/change-tracking-change-lists-web/src/main/java/com/liferay/change/tracking/change/lists/web/internal/menu/CTIndicatorProductNavigationControlMenuEntry.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+		sb.append("<link data-senna-track=\"temporary\" href=\"");
+
+		AbsolutePortalURLBuilder absolutePortalURLBuilder =
+			_absolutePortalURLBuilderFactory.getAbsolutePortalURLBuilder(
+				httpServletRequest);
+
+		sb.append(
+			absolutePortalURLBuilder.forModule(
+				_bundle, "/css/change_lists_indicator.css"
```
Comment (jbalsas) [inline modules/apps/change-tracking/change-tracking-change-lists-web/src/main/java/com/liferay/change/tracking/change/lists/web/internal/menu/CTIndicatorProductNavigationControlMenuEntry.java]: This makes me a bit uneasy... global css usually has unintended consequences... can you elaborate on why you need this?

File: modules/apps/change-tracking/change-tracking-change-lists-web/src/main/resources/META-INF/resources/css/change_lists_indicator.scss [hunk 2]
```diff
@@ -0,0 +1,187 @@
+$blue-color: #2e5aac;
+$green-color: #28a745;
+
+.change_tracking-control-group {
```
Comment (jbalsas) [inline modules/apps/change-tracking/change-tracking-change-lists-web/src/main/resources/META-INF/resources/css/change_lists_indicator.scss]: There's a lot of global overriding here... can you explain why is this needed? I'll involve someone on the team to assess if we can do this in a different way.

**Comment by stsquared99** [comment]
@jbalsas Thanks for getting back. Let me go over the existing code. Even though it was already there, I should be able to make some improvements to it, and then perhaps you and your team can provide further input. I will resend. Thanks!

**Pull request closed without merging**